Documentation: Add Building.md and walletconsole/README.md. (#747)
trustwallet/wallet-core: README.md

![Cocoapods](https://img.shields.io/cocoapods/v/TrustWalletCore.svg)
![Cocoapods platforms](https://img.shields.io/cocoapods/p/TrustWalletCore.svg)

> ## Trust Wallet Documentation

Reviewer: ```suggestion
## Documentation
```

See also [developer.trustwallet.com](https://developer.trustwallet.com) for more complete documentation.

## Supported Blockchains

We support Bitcoin, Ethereum, Binance Chain and 50+ blockchains, you can see the full list [here](docs/coins.md).

## Building

> For build instructions, see [Building.md](Building.md).

Reviewer: ```suggestion
For build instructions, see [Building.md](https://developer.trustwallet.com/wallet-core/building).
```

## WalletConsole Utility

Our project comes with an interactive command-line utility called _WalletConsole_, for accessing key and address management functionality of the library.  It can be started using:

```
$ ./build/walletconsole/walletconsole 
Wallet-Core Console                          (c) TrustWallet
Type 'help' for list of commands.
> help
```

> See the walletconsole [README](walletconsole/README.md) for details.

Reviewer: ```suggestion
See the walletconsole [README](https://developer.trustwallet.com/wallet-core/walletconsole) for details.
```
